src/pages/workspace/WorkspaceProfileSharePage.tsx

```diff
@@ -63,8 +72,29 @@ function WorkspaceProfileSharePage({policy}: WithPolicyProps) {
                     title={translate('common.share')}
                     onBackButtonPress={Navigation.goBack}
                 />
-                <ScrollView style={[themeStyles.flex1, themeStyles.pt2]}>
+                <ScrollView style={[themeStyles.flex1, themeStyles.pt3]}>
                     <View style={[themeStyles.flex1, isSmallScreenWidth ? themeStyles.workspaceSectionMobile : themeStyles.workspaceSection]}>
+                        <View style={[themeStyles.mh5]}>
+                            <Text style={[themeStyles.textHeadlineH1, themeStyles.mb2]}>{translate('workspace.common.shareNote.header')}</Text>
+                        </View>
+                        <View style={[themeStyles.mh5, themeStyles.mb9]}>
+                            <Text style={[themeStyles.textNormal]}>
+                                {translate('workspace.common.shareNote.content.firstPart')}{' '}
+                                <TextLink
+                                    style={themeStyles.link}
+                                    onPress={() => {
+                                        if (!adminRoom?.reportID) {
+                                            return;
+                                        }
+                                        Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(adminRoom.reportID));
```

Comment:
The navigation logic from this line led to the following issue:
- https://github.com/Expensify/App/issues/52258

because the RHP modal was not being closed before navigating. We fixed the issue by using `Navigation.dismissModal()` like so:

```ts
Navigation.dismissModal(adminRoom.reportID);
```

which does both, dismiss the modal and then navigate to the room.